pkg/resource/events-logger-config/reconciler.go

```diff
@@ -62,20 +45,27 @@ func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Inter
 	}
 
 	// Get desired config
-	desiredGrafanaAgentConfig, err := GenerateGrafanaAgentConfig(lc, currentApp.Spec.Namespace)
+	values, err := generateGrafanaAgentConfig(lc, r.DefaultWorkloadClusterNamespaces)
 	if err != nil {
 		logger.Info("grafana-agent-config - failed generating grafana-agent config!", "error", err)
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
+	desired := v1.ConfigMap{
```

Comment:
why this change compared to the main PR ?

Reply:
That's roughly what you did in main right? This is just doing the grafana agent config. How would you have done differently?

Reply:
I renamved the variable to use your name :)

Reply:
I think I'll still eventually rework some parts in the main one but for now I'm fine with this :)

Reply:
Oh I'm fine with you doing that :) I mostly wanted to be able to generate the templates to see why it's failing :)